packages/big-design/src/components/GucciModal/styled.tsx

```diff
@@ -0,0 +1,111 @@
+import { colors } from '@bigcommerce/big-design-theme/src/system/colors';
```

Comment:
Note: I imported `colors` from  here ^, since I wasn't able to use the `theme colors` from stitches inside the `rgba() fn`